android/src/com/mapswithme/maps/editor/MultilanguageAdapter.java

```diff
@@ -85,12 +57,35 @@ public Holder(View itemView)
     {
       super(itemView);
       input = (EditText) itemView.findViewById(R.id.input);
+      input.addTextChangedListener(new TextWatcher()
+      {
+        @Override
```

Comment:
Do you need these dummy overrides? Or beforeTextChanged in TextWatcher is abstract?